modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/CommunityIdProcessorTests.java

```diff
@@ -369,7 +371,12 @@ private void testCommunityIdProcessor(Map<String, Object> source, int seed, Stri
         );
 
         IngestDocument input = TestIngestDocument.withDefaultVersion(source);
-        IngestDocument output = processor.execute(input);
+        IngestDocument output;
+        try {
+            output = processor.execute(input);
+        } catch (Exception e) {
+            throw ExceptionsHelper.convertToRuntime(e);
```

Comment:
I did not realize until this moment that ExceptionsHelper.convertToRuntime was compatible with expectThrows.